gbpservice/nfp/orchestrator/drivers/orchestration_driver.py

```diff
@@ -149,35 +169,169 @@ def _create_management_interface(self, device_data, network_handler=None):
 
     def _delete_management_interface(self, device_data, interface,
                                      network_handler=None):
-        try:
-            token = (device_data['token']
-                     if device_data.get('token')
-                     else self.identity_handler.get_admin_token())
-        except Exception:
-            self._increment_stats_counter('keystone_token_get_failures')
-            LOG.error(_LE('Failed to get token for management interface'
-                          ' deletion'))
-            return None
+        token = self._get_token(device_data.get('token'))
+        if not token:
+            return False
 
         network_handler.delete_port(token, interface['id'])
 
+    def _delete_interface(self, device_data, interface,
+                          network_handler=None):
+        token = self._get_token(device_data.get('token'))
+        if not token:
+            return False
+
+        network_handler.delete_port(token, interface['id'])
```

Comment:
delete_port needs to be defined in coal neutron network driver also, as it is defined in coal gbp network driver for uniformity. This is needed in case of neutron workflow.